### ISSUE TITLE
Model maker shouldn't look like a link

### ISSUE THREAD

**Issue by bbondy** [brave/brave-browser#34258]
I think this is just removing the bold on Meta in "llama2 13b by Meta"?
<img width="320" alt="Screenshot 2023-11-15 at 4 28 59 PM" src="https://github.com/brave/brave-browser/assets/831718/99de014c-800d-472e-8e4f-449b7a29d5aa">

**Comment by aguscruiz** [comment]
Also use the same color as the other intro text.
Design here:
https://www.figma.com/file/h7fKSJF2P556bZWfyLTfwD/Leo-MVP?type=design&node-id=3251%3A39646&mode=design&t=IdjTvssS98ypNYmE-1

![image](https://github.com/brave/brave-browser/assets/15734630/677b5a66-da6f-4d2f-acbe-b7a0ea65a1b1)

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

```
Brave | 1.63.51 Chromium: 120.0.6099.115 (Official Build) nightly (64-bit)
-- | --
Revision | d4b258be9973bf7f91ae4f1c7ca2eea5c2882085
OS | Windows 10 Version 22H2 (Build 19045.3803)
```

### Confirmed the `Meta` in `[llm-name] by Meta` is no longer bolded, and is the same color and font weight as the rest of its byline text

`llama2-13b` | `llama2-70b` | `Claude Instant`
----------|----------|-----------
![image](https://github.com/brave/brave-browser/assets/387249/100464a9-15c6-439d-af51-9f39fbd378f3) | ![image](https://github.com/brave/brave-browser/assets/387249/c8269d7e-6de8-47f8-b1fb-e420083d8caa) | ![image](https://github.com/brave/brave-browser/assets/387249/ea425ff8-19de-493f-8974-0caa715bfdca)

**Comment by LaurenWags** [comment]
@nullhook @bbondy @rebron could someone please add the appropriate release notes label for this one?

**Comment by kjozwiak** [comment]
Removing `QA Pass` labels as the above will need to be re-checked using `1.62.x`. However, used https://github.com/brave/brave-browser/issues/34258#issuecomment-1861948200 as the needed verification to get the above uplifted into `1.62.x`.

Adding `QA/Blocked` for now as the above was merged into `cr121-1.62.x` via https://github.com/brave/brave-core/pull/21593. Once `C121` is merged into `1.62.x` via https://github.com/brave/brave-core/pull/21593, we'll unblock the above and mention the needed `1.62.x` version for verification 👍

**Comment by kjozwiak** [comment]
The above requires `1.62.148` or higher for `1.62.x` verification 👍

**Comment by stephendonner** [comment]
## Verified `PASSED` using

```
Brave | 1.62.149 Chromium: 121.0.6167.75 (Official Build) (x86_64)
-- | --
Revision | ff84587bd70af9fcbcbe59fc5194ca65082759c4
OS | macOS Version 11.7.10 (Build 20G1427)
```

`Mixtral` | `Claude Instant` | `Llama2-13b`
---------|------------------|--------------
<img width="325" alt="Screen Shot 2024-01-22 at 1 16 18 PM" src="https://github.com/brave/brave-browser/assets/387249/a1785041-23d4-40e9-99c0-15679292b17d"> | <img width="325" alt="Screen Shot 2024-01-22 at 1 17 19 PM" src="https://github.com/brave/brave-browser/assets/387249/cd846fb9-9297-49e0-8109-8b8c2068d98a"> | <img width="323" alt="Screen Shot 2024-01-22 at 1 17 33 PM" src="https://github.com/brave/brave-browser/assets/387249/974c578b-65e5-4844-b500-fb71e7d3a297">
